tests/bsim/bluetooth/samples/battery_service/prj.conf

```
@@ -0,0 +1,18 @@
+CONFIG_BT=y
+CONFIG_LOG=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+CONFIG_BT_SMP=y
+CONFIG_BT_SIGNING=y
+CONFIG_BT_BAS=y
+CONFIG_BT_ATT_PREPARE_COUNT=2
+CONFIG_BT_GATT_CLIENT=y
+CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
+CONFIG_BT_DEVICE_NAME="bsim_bas"
```

```suggestion
CONFIG_BT_BAS=y
CONFIG_BT_GATT_CLIENT=y
CONFIG_BT_DEVICE_NAME="bsim_bas"
```

These seem unnecessary